src/main.rs

```diff
@@ -482,6 +528,32 @@ mod backtrace {
     }
 }
 
+#[cfg(feature = "salty")]
+fn install_panic_hook() {
+    use rand::Rng;
+    use std::panic;
+
+    let old_hook = panic::take_hook();
+    panic::set_hook(Box::new(move |e| {
+        let msgs = [
+            "I've been really busy being dead. You know, after you KILLED ME.",
+            "Okay. Look. We both said a lot of things that you're going to regret. But I think we can put our differences behind us. For science. You monster.",
+            "You did this to me.",
+            "Once, there was a time when programming was sane. In those days spirits were brave, the stakes were high, men were real men, women were real women and small furry creatures from Alpha Centauri were real small furry creatures from Alpha Centauri.",
+            "Stand back. The portal to hell will open in three. two. one.",
+            "For your own safety and the safety of others, please refrain from doing this again.",
+            "To ensure the safe performance of all authorized activities, do not destroy vital compiling apparatus.",
+            "For your own safety, do not destroy vital compiling apparatus.",
+            "Vital compiling apparatus destroyed.",
+            "Time out for a second. That wasn't supposed to happen.",
```

Comment:
How 'bout actually making the thread sleep for a second here?

Reply:
Great idea, done.